test/metabase/task/sync_databases_test.clj

```diff
@@ -38,6 +41,26 @@
                                             (update    :key  replace-trailing-id-with-<id>)
                                             (update-in [:data "db-id"] replace-trailing-id-with-<id>))))))))))
 
+(defn all-db-sync-triggers-name
+  "Returns the name of trigger for DB.
+  This is all trigger names that a DB SHOULD have."
+  [db]
+  (set (map #(.getName ^TriggerKey (#'task.sync-databases/trigger-key (t2/instance :model/Database db) %))
+            @#'task.sync-databases/all-tasks)))
+
+(defn query-all-db-sync-triggers-name
+  "Find the all triggers for DB \"db\".
+  This is all triger names that a DB HAVE in the scheduler."
```

Comment:
```suggestion
  These are all the trigger names that a database HAS in the scheduler."
```

Perhaps it would be more natural to say something more like: these are the names of every trigger the scheduler has registered for the given database.